vita3k/renderer/src/state_set.cpp

```diff
@@ -106,7 +106,7 @@ COMMAND_SET_STATE(uniform_buffer) {
     switch (renderer.current_backend) {
     case Backend::OpenGL: {
         gl::set_uniform_buffer(*reinterpret_cast<gl::GLContext *>(render_context), is_vertex, block_num, size, data, config.log_active_shaders);
-        delete data;
+        delete[size] data;
```

Comment:
This is not valid C++.
MSVC probably accepted it because MSVC is dumb and accepts all sorts of non-compliant code.
```suggestion
        delete[] data;
```